submissions/jeffersonMormah/introductionToHtml/index.html

```diff
@@ -0,0 +1,67 @@
JepIwGKCcGcXb0rN7dfeNdWvQvY509TL/AGofVkPsG+6K5tmuXub4L6UO90UsuG6KkCubcIzakHjhf8Ui7shtOozBzTQ8KxM3oqW1ktce5w+TUcFo96m1QTgxc5uMuhZLW7vxkEniOes0lhmhPI6WtSfNODzEGqpAyOmOrUSVEk9dWiiMES+84OCiPVT4oXLAxfdcXnfWpXrUTTEhbGavO0pqQpkjaR9MgnpFBRn0BsmrMFsE8EAU9CCwYFSAYoAze7D61m9uvvGurXoXsc+epl/tPouH2CO6K5tmuXuboaUO9KWWCgCB2xTmznqWP5pV2kbTqMtd849Gaws3oo224xdYXQWjj9VaaNDEXakWi06w289FY59zTHsOXBgHFVJI2VvfaNMiVZFP51yqnIWxm4OumIWzwxrVxbJK2f1E+ugqzedkz9CBTkILNUkAaCTNrt6Vm9uvvGutXoXsc+epl/tXoyH2CO6K5tmuXuboaUOqWWCgCH2rTvWZzqUKVbpGVajK3x9Iaws3xKTt4kJmW1fSlwewp8a0cO8xYm7Uix2jHxJo9XGsk+5qXYcO5NVJI2UMZ40yJRkU/T0LY0cAAq6Fs8MfOpiFktaEAyWgedYqSrN02U/p5zTTOWagDlBJm929Kze3X3jXWr0L2OfPUy/2r0ZD7BHdFcyzXL3N0NKHVULBQBF7SDes7/Vg++l26S9WoymWkhzSsMjoRKbt8Po7crodWPcPCm8N+YVf6E5Zz/2LWn9tZrO5oXYcugHjVUWIySAc83tpsSjIt8U5C2NXBpV0LZ5JHzquUJGCsNutknB3hVoipPobhsQVKiJKuemiS2A0ABNBJm12P1rN7dfeNdavQvY589TL9avRcPsEd0VzLNcvc2w0odVQsdoAaz44lRXWFcFpx6qrJZWC0Xh5MpukZyLLdYfTuuIOCP59Vc+Sw8M6EGmslJ2/QBCgq5/jGP8AafCm8N3ZS/8AKTNnA8ntH/xrNPUaV2Pd1OhqpJGShxpsSjIx5OTTkLY2WkAVdFGeaRrVkKJ3ZezSLzckhpB5BkgrXzZ5hTYiZs3axQkw4yUDoqwslakAoJM3u3pWb26+8a61ehexz56mX61ei4fYI7ormWa5e5thpQ6qhYKAEqGeNAEXerJFu7QDwKHU6IdTxT4jqpdlamMrscOxnm0/wdT7smNEbU2lCH+UMgHQDBHDp14UmquUJMdZapJHsdiLhbIyW2cSkJGN5vQ+w/xSbKZZyOhfB9GQ8uG8woodbW2voWnBpPK13HqSZEyY7pJxk0yKKtjB2I7zg01C2xDFnmznOThxnn19DaSqrpMo2kW2wfBjNkLS7e3RFZ48i0QpxXUTwT7/AMqaoszysXoabarREtrCI8JhLTSNAEim4wK7kuhIAFBAqgkDQQZvdvSszt19411a9C9jBPUy/Wr0XD7BHdFc2zXL3NsNKHVULBQAUAGKAAigBJTQSJU0lad1YCh0EZqGkwTwNl2q3rOVw45PWymo5Y7E8z3G8yNabZEcluxWEIbx5rIyok4CRpqSSABzkip5UHM2Rn1lKbLkmZ5NZ/saYUhAT1KcUFbx/wBIAHDJ4ligUczoNxgNokR5XlOOThTSyhS1dSHEhI3tOChqcDeHPDiwyiwQn2JcVqTFWFsvIC0K4ZBqpYhJt2kSXFJhvpiw0Olnlw1yrshwZBQ0nhodN451BGOegBuFzGCkm63aK45gNquLDS2So8ArdCSPUFCgkm7ZPXJW9GltBidHxyzQVlJB81aTzpOD+YI5qCCjXb0rN7dfeNdavQvY589TCJ8JsOPFZYVapii02lBIW3g4GMjWlT/D73JtL+Qhx9Cik3/B7fKnC/CZv62/GqeXX7L5LeYUbv4D5VIX4TN/W340eXX7L5DzCjd/AfKpC/CZv62/Gjy6/ZfIeYUbv4D5VIX4TN/W340eXX7L5DzCjd/AfKpC/CZv62/Gjy6/ZfIeYUbv4D5VIX4TN/W340eXX7L5DzCjd/B35VIX4RN/W340eXX7L5DzCjd/AfKpC/CJv62/Gjy6/ZfIeYUbv4PA7cMbQ3W1wGocmMn4wt1anClQVusuFI0PMoJPrAqlnCW1YlPsMq4uu1uMO543G73QPy0sqLqGJqmmgqIUnkEoyClW7jztNeO7oda0Rqrws+q39c7FHOWen9C4t4vZeaQqM42t5bCVR1wlI5RlTe8t0q13SFc2dOFEqqsZzv6+ueiIU57fx6bnV7ds2FdztyYT7zjch1TbiCkJSpQ3iME50WVaVmhwltkeeK6DLOLqrlyTfUnWIKm7XDftzilFFoQ3EA144KlA86iMf4a5fFTn4clX3x0OrwsIeJF2ds9Tws7YeQ/FYakuNPMrMtx7I3TukBKevOueNc/gHizlgnh92/6Oj+IpOHNNrK7Jf2/8Gtw2iFgj7OXGYw8/Ift62nm0kJWdGyCc9Bz7TXfqpndLlh3PP3XQqjzTKtL2qYky3nxFfSHXFL3SpORk5xxrsw4WxRSZyJ8bW5MqldE5QUAFABQAUAFABQAUAFAC2XnI7zb7C9x1pQWhY/tI1FKuqV0HBjqLXVYpo0pnb603KE35SacjT06LTkhtXSUqwRjqVg+zNcF8NbVLEl03xk78eJptjlP9s4EztvbbBtrjdoaXJnugjLmrTeecnACvUka89TDhbrpdF037f2RPiqaY5b67dzNVKUtSluLUtxZKlrUdVKJySevNd6quNcFCPZHn7bJWTc5epeNidto9ritW29trVFYOY0hCCstD7JSNcDmIzppXE4z8Pkpc8Flf0dzg/wAQi48s3h/2WS/bYbLvstupusla0A7jUNtW8onGhynA4DiRWONFsniMX8GyV9UVmUkZnfbvIvU1MiQClDbYaYbJyUNjpPOo8SfCu7wXCeBHMu7OFx3Fq+WI9kR1bjAFAHKACgAoA7QBznoA7QBwUAFAAKAFHjQSB41JAmgBXNUEgeaoyyX2OVJUKAP/2Q=="/>
```

Comment:
Well done @Jeffbuchi 

Here, instead of making use of a `base64` image as you did, I think you should create a `images` folder in your `introductionToHtml` folder. Place download and place the images you want to display into the `images` folder.

So, here the `src` attribute will then point to it like this `src="images/image_name.extension"`.

I hope this helps

Reply:
Thanks ma'am.

Reply:
Thanks. It has been refactored just now